xml/System/Uri.xml

```diff
@@ -618,7 +618,9 @@ http://myUrl/
   
 ## Remarks  
  This constructor creates a <xref:System.Uri> instance by combining the `baseUri` and the `relativeUri`. If `relativeUri` is an absolute URI (containing a scheme, host name, and optionally a port number), the <xref:System.Uri> instance is created using only `relativeUri`.  
-  
+ 
+ If the `baseUri` has relative parts (like `/api`), then the relative part must be terminated with a slash, (like `/api/`). 
```

Comment:
It would be useful to add the condition here: "...if the relative part is to be preserved in the constructed \<xref:System.Uri>." That's true of the second overload as well.